modules/identity/src/Volo.Abp.Identity.EntityFrameworkCore/Volo/Abp/Identity/EntityFrameworkCore/EfCoreIdentityUserRepository.cs

```diff
@@ -263,6 +263,25 @@ select ouRole.RoleId
             .LongCountAsync(GetCancellationToken(cancellationToken));
     }
 
+    public async Task<List<RoleWithUserCount>> GetCountAsync(Guid[] roleIds, CancellationToken cancellationToken = default)
```

Comment:
This method is not performant. Should be revisited.

Reply:
https://github.com/abpframework/abp/pull/17380

Reply:
> This method is not performant. Should be revisited.

https://github.com/abpframework/abp/pull/17380